src/sonic_ax_impl/mibs/ietf/rfc2737.py

```diff
@@ -37,10 +37,10 @@ class XcvrInfoDB(bytes, Enum):
     """
 
     TYPE              = b"type"
-    HARDWARE_REVISION = b"hardwarerev"
-    SERIAL_NUMBER     = b"serialnum"
-    MANUFACTURE_NAME  = b"manufacturename"
-    MODEL_NAME        = b"modelname"
+    HARDWARE_REVISION = b"hardware_rev"
+    SERIAL_NUMBER     = b"serial"
+    MANUFACTURE_NAME  = b"manufacturer"
+    MODEL_NAME        = b"model"
```

Comment:
It is better to move into a common place and shared with the repo sonic-platform-common.

Reply:
It would prevent the need for dependent changes like this in the future. Please feel free to open an issue.

Reply:
Could you clarify "prevent the need for dependent changes like this in the future"? I think it will make it easier.

Reply:
This PR was necessary because the key names were updated in sonic-platform-common. If the two repos shared a common source for the data, this PR would not be necessary.